os/board/artik05x/userspace/Makefile

```diff
@@ -94,6 +94,7 @@ OBJS  = $(COBJS)
 # Targets:
 
 all: $(USER_BINPATH)$(DELIM)tinyara_user.elf $(USER_BINPATH)$(DELIM)User.map
+	$(Q) rm -f tinyara_user.elf
```

Comment:
Let's use `DELFILE` instead of `rm -f`. Actually both are same but I want to keep same usages for Makefile.

Reply:
OK. I will update

Reply:
But I don't know this is necessary or not. After getting approval from @kishore-sn , I will merge this.

Reply:
OK